portfolio/src/main/webapp/script.js

```diff
@@ -68,6 +72,11 @@ async function loadComments(limit = -1, admin=0) {
   for (let i = commentsJson.length - limit; i < commentsJson.length; i++) {
     commentsJson[i] = JSON.parse(commentsJson[i]);
 
+    if (language != 'original') {
+      commentsJson[i].comment =
+      await translate(commentsJson[i].comment, language);
```

Comment:
Could you please add a TODO on limiting the max size of the text area? As discussed before, it will not completely stop  potential hackers from spending your intern limit, but at least will make some troubles :)

Reply:
I fixed it by limiting the textarea field with 100 character.